c2cgeoportal/scaffolds/__init__.py

```diff
@@ -72,6 +73,27 @@ def pre(self, command, output_dir, vars):
 
         return ret
 
+    def _set_srid_in_vars(self, command, vars):
```

Comment:
srid is now used both in create and update scallfold. So it is not just for fun except if you think that there is another way to use this method in two different class.